app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -128,7 +128,10 @@ import kotlinx.coroutines.CoroutineScope
 
 @Composable
 fun PostHeaderLine(
-    postView: PostView,
+    community: Community,
```

Comment:
Whats the reason to break out postView? I couldn't find why this was necessary below.

Reply:
We used pass the entire postview into this composable but we use only select parts from it.

For example we have `pv1 = PostView(...)`, we update our vote on it `pv2 = pv1.copy(my_vote=1)` 
Now `pv1 ~= pv2` This composable now recomposes as they are not equal (Stable are compared on equality)
But this Composable doesn't use `my_vote` But it still has to unnecessary recompose  because it arguments are different. Now Since `Post`, `Community` etc stayed the same `pv1.Post == pv2.Post` This composable arguments did not change and thus didn't need to recompose.

Btw Strong skipping made it so that unstable are compared on reference. If their ref is the same they won't recompose. Before all unstable would always recompose. This also means for example that list (which is still considered unstable) will no longer recompose. (We have marked it as stable, because we treat as such)  

If you watch the recompositions (in the layout inspector) for postlisting viewmode then you will see that only the buttons recompose when you upvote or downvote. As they are the only composables that use `my_vote`.

Reply:
Ahh mmk, makes sense.